src/bin/stratisd.rs

```diff
@@ -7,16 +7,16 @@
 #![cfg_attr(not(feature = "clippy"), allow(unknown_lints))]
 #![allow(doc_markdown)]
 
-extern crate devicemapper;
-extern crate libstratis;
-#[macro_use]
-extern crate log;
 extern crate clap;
 #[cfg(feature = "dbus_enabled")]
 extern crate dbus;
+extern crate devicemapper;
```

Comment:
All this is just a change of ordering....perhaps rustfmt did it. I would think that either it is a mistake or it belongs in a separate PR.

Reply:
New rustfmt does sort `extern`, `use` and some other things.

Reply:
Those are `rustfmt` changes

Reply:
That's what I figured, somehow. But I don't think it belongs in this PR if it is just a re-ordering.

Reply:
What happened here? Did our mass rustfmt conversion PR miss this file?

Can we just merge this and get on with it?

Reply:
I think what happened is a difference in rustfmt versions.  This PR has spanned the change in versions and I think an older version did this as I reverted this change and the current rustfmt did not mess with it